### PR TITLE
chore: add Prettier w/ lint-stage configuration

### PR DESCRIPTION
**Pull request by KyleAMathews**

### CODE REVIEW

File: examples/google-auth/README.md [hunk 1]
```diff
@@ -7,27 +7,25 @@
...
 
     - `GOOGLE_CLIENT_ID`: Fetch the ClientId from the newly created OAuth 2.0 Client ID. Yuo can find it in the [Google Credentials Console](https://console.cloud.google.com/apis/credentials)
     - `GOOGLE_CLIENT_SECRET`: Fetch the Client Secret from the newly created OAuth 2.0 Client ID. Yuo can find it in the [Google Credentials Console](https://console.cloud.google.com/apis/credentials)
 
-    You'll also want to add these as environment variables when deploying to Gatsby Cloud. 
-
+    You'll also want to add these as environment variables when deploying to Gatsby Cloud.
 
-2.  **Start developing.**
+3.  **Start developing.**
```
Comment (dmccraw) [inline examples/google-auth/README.md:28]: I noticed a readme above was converted to 1. for all of the steps? Was that intended and do we want to change all the readme's?

File: examples/google-auth/src/api/labels.js [hunk 2]
```diff
@@ -1,67 +1,67 @@
-const url = require("url");
-const { google } = require("googleapis");
+const url = require("url")
+const { google } = require("googleapis")
 
-const clientId = process.env.GOOGLE_CLIENT_ID;
-const clientSecret = process.env.GOOGLE_CLIENT_SECRET;
+const clientId = process.env.GOOGLE_CLIENT_ID
+const clientSecret = process.env.COOGLE_CLIENT_SECRET
```
Comment (dmccraw) [inline examples/google-auth/src/api/labels.js:5]: `GOOGLE_CLIENT_SECRET`?

File: examples/google-auth/src/api/login.js [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
 
-const { google } = require("googleapis");
-const people = google.people("v1");
+const { google } = require("googleapis")
+const people = google.people("v1")
 
-const clientId = process.env.GOOGLE_CLIENT_ID;
-const clientSecret = process.env.GOOGLE_CLIENT_SECRET;
+const clientId = process.env.GOOGLE_CLIENT_ID
+const clientSecret = process.env.COOGLE_CLIENT_SECRET
```
Comment (dmccraw) [inline examples/google-auth/src/api/login.js:12]: GOOGLE_CLIENT_SECRET?

File: examples/google-auth/src/api/labels.js (hunk 2 above)
Comment (KyleAMathews) [reply examples/google-auth/src/api/labels.js:5]: Hmm I fixed that already. not sure why it came back.

File: examples/google-auth/src/api/logout.js [hunk 4]
```diff
@@ -1,32 +1,32 @@
-const { google } = require("googleapis");
-const clientId = process.env.GOOGLE_CLIENT_ID;
-const clientSecret = process.env.GOOGLE_CLIENT_SECRET;
+const { google } = require("googleapis")
+const clientId = process.env.GOOGLE_CLIENT_ID
+const clientSecret = process.env.COOGLE_CLIENT_SECRET
```
Comment (dmccraw) [inline examples/google-auth/src/api/logout.js:3]: GOOGLE_CLIENT_SECRET?

File: examples/twilio-text/README.md [hunk 5]
```diff
@@ -37,11 +37,11 @@
 
     Edit `src/pages/index.js` to see your site update in real-time!
 
-3. **Deploy**
+4.  **Deploy**
```
Comment (dmccraw) [inline examples/twilio-text/README.md:40]: 3.?

File: examples/google-auth/README.md (hunk 1 above)
Comment (KyleAMathews) [reply examples/google-auth/README.md:28]: I don't think so. Lemme check locally.

**Review by dmccraw — APPROVED** [review]
:ship: it!

File: examples/twilio-text/README.md (hunk 5 above)
Comment (KyleAMathews) [reply examples/twilio-text/README.md:40]: no it was wrong before — open up the folded code, there's a 3 there.

File: examples/google-auth/README.md (hunk 1 above)
Comment (KyleAMathews) [reply examples/google-auth/README.md:28]: fixed. Not sure why prettier did that.

**Comment by KyleAMathews** [comment]
Thanks for the review!

**Pull request merged**